### DISCUSSION TITLE
Can not make it work on Asahi Linux

### DISCUSSION THREAD

**Discussion by qstrql** [prasanthrangan/hyprdots#464] [Q&A]
Hi, I tried installing hyperdots on my m2 macbook air running asahi arch linux and it did not work. is it impossible to run or do i need to tweak some things ? thanks

**Comment by kRHYME7** [comment]
Hello what errors you got?

**Reply by prasanthrangan** [reply]
Please refer #394 , @bouyagas was able to make it work on M1 chip!
